cypress/integration/App.spec.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+describe('App >', () => {
```

Comment:
Should this be `<App >`?

Reply:
I all make that `App component`. So that it reads well with its test.

Like.. `App component -> should have length 3`